topics/imaging/tutorials/omero-suite/tutorial.md

```diff
@@ -0,0 +1,376 @@
+
+
+## Before Starting - Set up the OMERO credentials
+Before starting you need to set up your OMERO credential. 
+This is necessary to connect to a target OMERO instance and needs to be performed just one time.
+
+The OMERO credentials have to be saved in your preferences 
+(*User → Preferences → Manage information*).
+
+![omero_credential.png](../../images/omero-suite/omero_credential.png)
+
+This feature is embedded in Galaxy and needs to be enabled by the admin 
+of the server you are using. You can now use it in [UseGalaxy.eu](https://usegalaxy.eu/) or ask your 
+admin to add it. For more info about the configuration on the admin side, 
+please take a look at the [README file of the tool](https://github.com/galaxyproject/tools-iuc/blob/main/tools/idr_download/README.md).
+
+**Please Notice**: The configuration file, which contains your OMERO password and username, will be stored in the job working directory.
+    This directory only exists during the runtime of the job and should only be accessible by the system user that runs the job.
+    However, please be aware that your username and password **may be exposed** to users with administrative rights.
+    We are working on increasing the security of the OMERO suite
+
+# Prepare your data
+
+We need a dataset to upload into your OMERO instance. 
+To this end, we have prepared an integrative dataset with images, metadata and regions of interest (ROIs) in Zenodo. 
+
+
+## Data Upload - Image, metadata and ROI files
+
+> <hands-on-title>Data Upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial in Galaxy.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the nuclei imaging data from [Zenodo](https://zenodo.org/records/14196675) or 
+> from the shared data library:
+>    - **Important:** Choose the correct data type if prompted.
+>
+>    ```
+>    https://zenodo.org/records/14205500
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>    
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+>   
+>    - Select the following files: 
+>    
+>       - `image85-H-01-00.tif`
+>       - `image85-H-01-00.tif.tabular`
+>       - `image86-H-02-00.tif`
+>       - `image86-H-02-00.tif.tabular`
+>       - `image87-H-03-00.tif`
+>       - `image87-H-03-00.tif.tabular`
+>       - `metadata_dataset.csv`
+> 
+> 3. Tag each dataset with a label like "image" (tif files), "rois" (tabular files) and "metadata" (CSV file)
+> for easy identification
+{: .hands_on}
+
+# Upload images and metadata
+
+In this section, we will focus on how uploading images and metadata into a user-defined OMERO instance.
+This is done by using the **OMERO Image Import** and **OMERO Metadata Import** tools.
+
+## Step 1: Image Upload
+
+> <hands-on-title>Upload images into OMERO</hands-on-title>
+>
+> 1. {% tool [OMERO Image Import](https://toolshed.g2.bx.psu.edu/view/ufz/omero_import/5.18.0+galaxy3) %} with the following recommended parameters:
```

Comment:
Those are tool IDs, they need to look like `toolshed.g2.bx.psu.edu/repos/bgruening/black_forest_labs_flux/black_forest_labs_flux/2024+galaxy2`

You can find this link in the tool menue
![grafik](https://github.com/user-attachments/assets/b771e5a2-6c51-4e04-9655-ef428f0a1f2a)

Reply:
This would need to be changed in all tools.